amo2kinto/exporter.py

```diff
@@ -137,35 +153,17 @@ def write_plugin_items(xml_tree, records, app_id, api_ver=3, app_ver=None):
 
     pluginItems = etree.SubElement(xml_tree, 'pluginItems')
     for item in records:
-        if is_related_to(item, app_id):
-            for versionRange in item.get('versionRange', []):
-                if not versionRange.get('targetApplication'):
-                    add_plugin_item(pluginItems, item, versionRange,
+        for versionRange in item.get('versionRange', []):
+            if not versionRange.get('targetApplication'):
```

Comment:
can't we use `is_related_to()` here?

Reply:
I tried this first, but it didn't work, which is why I factored out part of `is_related_to()`. The problem is you can invoke `add_plugin_item` both with and without a `targetApplication`. Iff you pass one, it includes information from that `targetApplication` into the bit it writes out into `blocklist.xml`. In other words, when I did that, the tests failed. :-)